src/core/services/convert.spec.js

```diff
@@ -0,0 +1,37 @@
+import * as Convert from './convert';
+
+describe('Convert', () => {
+	const parseText = Convert.parseFactory('text');
+	const parseCurrency = Convert.parseFactory('currency');
+	const parseNumber = Convert.parseFactory('number');
+	const parseDate = Convert.parseFactory('date');
+	const parseBool = Convert.parseFactory('bool');
+
+	describe('parseFactory', () => {
+		it('should parse Number to String', () => {
```

Comment:
1. for each parse[Type] add own describe,
2. for each parse[Type] add check to null and undefined
3. more tests per parse[Type] (eg. for parseNumber -> parseNumber('1.0), parseNumber('-1.2), parseNumber('-1e04') and more...)